test/testdata/configs/go-header.yml

```diff
@@ -1,6 +1,6 @@
 linters-settings:
   goheader:
-    template: MY {{title}}
+    template-path: go-header-template
```

Comment:
let's keep both tests
and `template` and `template-path`
(with better naming for config and test files)